.github/actions/check-dependant-rust-repo/action.yml

```
@@ -0,0 +1,125 @@
+name: Pull an outside repo that submodules the current repo, try to uprev the submodule and see if it still builds.
+
+inputs:
+    remoteRepo:
+      description: The username/repository-name of the repository to check
+      required: true
+    id:
+      description: A unique identifier for this step
+      required: true
+    githubToken:
+      description: A GitHub token for API access.
+      required: true
+    submodulePath:
+      description: The path to the submodule we are going to try upreving
+      required: true
+    buildCmd:
```

Optional/Equivalent: Spell out `Command`
```suggestion
    buildCommand:
```

---

Not sure what this gets us, cmd is a well established shorthand.